src/clients/api/queries/useGetVaults/constants.ts

```diff
@@ -0,0 +1,5 @@
+import { getToken, getContractAddress } from 'utilities';
+
+export const XVS_TOKEN_ID = 'xvs';
```

Comment:
These constants got duplicated a couple times. I moved them to the constants folder.